SportHub/src/modules/ArticleView/components/ArticleView.js

```diff
@@ -2,20 +2,26 @@ import React from "react"
 import styles from "../styles/style.module.scss"
 import { useState, useEffect } from "react"
 import * as DOMPurify from 'dompurify'
+import { useNavigate } from 'react-router-dom'
+import { ROUTES } from "../../../routes/routes"
 
 import getArticleByIdAndLanguage from "../helpers/getArticleByIdAndLanguage"
 
-function ArticleView({ articleId, language}) {
+function ArticleView({ articleId, language }) {
+    const navigate = useNavigate()
     const [article, setArticle] = useState(null)
 
     const getArticle = async () => {
         const result = await getArticleByIdAndLanguage(articleId, language)
+        if (!result.title) {
```

Comment:
this will work for error, but this isn't the clearest way to write it. I would suggest to console.log(error); return null and then check here if(result){setArticles...}